grades/signals.py

```diff
@@ -0,0 +1,20 @@
+"""
+Signals for micromasters course certificates
+"""
+from django.db import transaction
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from grades.models import MicromastersCourseCertificate
+from grades.api import generate_program_certificate
+
+
+@receiver(post_save, sender=MicromastersCourseCertificate, dispatch_uid="coursecertificate_post_save")
+def handle_create_coursecertificate(sender, instance, created, **kwargs):  # pylint: disable=unused-argument
+    """
+    When a MicromastersCourseCertificate model is created
+    """
+    if created:
```

Comment:
Can you add a test for the case that you resave the `MicromastersCourseCertificate` and `generate_program_certificate` is not called`?